generate/generate.go

```diff
@@ -316,7 +316,8 @@ func Generate(config *Config) (map[string][]byte, error) {
 	// way.  (As-is, we generate a broken file, with just (unused) imports.)
 	if len(document.Operations) == 0 {
 		// Hard to have a position when there are no operations :(
-		return nil, errorf(nil, "no queries found, looked in: %v",
+		return nil, errorf(nil,
+			"no queries found, looked in: %v (configure this in genqlient.yaml)",
```

Comment:
operations are strings, so changing to `%+v` is useless at present, but if we ever pivot to making them structs, we adding the harmless `+` would make it Just Work™️

Reply:
I'm not sure if I see why we would make this a struct(s)?  Or rather, if we did, we would want to change the error to still just print the filenames.  (The snapshot-test will help us remember!)